crates/web-sys/webidls/enabled/RTCRtpSender.webidl

```diff
@@ -51,28 +51,26 @@ dictionary RTCRtcpParameters {
   boolean   reducedSize;
 };
 
-dictionary RTCRtpCodecParameters {
-  unsigned short payloadType;
-  DOMString      mimeType;
-  unsigned long  clockRate;
-  unsigned short channels = 1;
-  DOMString      sdpFmtpLine;
+dictionary RTCRtpCodec {
+  required DOMString     mimeType;
+  required unsigned long clockRate;
```

Comment:
```suggestion
  DOMString     mimeType;
  unsigned long clockRate;
```
I believe both of these can't be required, because it changes the `new()` function of `RTCRtpCodecParameters`, which is a breaking change.